Weekly progress in development

Pla5ma opened the pull request:
**readme.me**
- Updated to reflect changes in scripts Get-AvailabilityInformation.ps1 and Get-Region.ps1

**Get-AvailabilityInformation.ps1**
- Mapping of a customer implementation from 1-Collect script to Azure regions
- Fully working
- SKU mapping working for VM SKUs and VM disk SKUs
- Next step: General SKU mapping implementation

**Get-Region.ps1**
- Mapping if script output from script Get-AvailabilityInformation.ps1 to a specific selected region
- Fully working

jfaurskov commented on 2-AvailabilityCheck/readme.md:
Please include a comment to the effect that currently the services supported by this script are fairly limited as well as which services are covered.

jfaurskov commented on 2-AvailabilityCheck/Get-AvailabilityInformation.ps1:
Script needs to take Global services into account. A service like microsoft.insights/actiongroups is global and will show up as false for each separate region in the output file which is not correct since it's basically available everywhere.

jfaurskov reviewed (changes requested):
Had a couple of comments, otherwise looking good.

Pla5ma commented:
New topics added:
- Proper handling of logical region 'Global', e.g. will not be displayed anymore in result but corresponding physical regions will be set to available
- Added more detail information in readme.md about currently supported SKUs
- Added improved error handling with detailed information; still some parts missing
- Implemented SKU retrivel for Microsoft.Sql/managedInstances and Microsoft.Sql/servers/databases; mapping missing
- Changed .gitignore to include all result files by using wildcard

jfaurskov reviewed (approved):
LGTM
